rhv_checks.py

```diff
@@ -338,6 +340,55 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the host's Hosted Engine Status."""
+    okay, critical, unknown, over_all = [], [], [], []
+    command = "hosted-engine --vm-status --json"
+    hosts = system.api.system_service().hosts_service().list()
+
+    for host in hosts:
```

Comment:
From what I can tell, there is no need to run this command on each host, since it outputs information about every host after running.  So you could choose e.g. the first host from the list of hosts to run this command on.

Reply:
Correct. I will use single host and iterate with all hosts.

Reply:
Updated test with required changes.